src/main/java/org/kohsuke/github/GHPullRequest.java

```diff
@@ -329,51 +339,57 @@ public GHPullRequestReviewComment createReviewComment(String body, String sha, S
                 .to(getApiRoute() + "/comments", GHPullRequestReviewComment.class).wrapUp(this);
     }
 
+    /**
+     * Create a reply to the current comment.
+     */
+    public GHPullRequestReviewComment createReviewCommentReply(GHPullRequestReviewComment comment, String body) throws IOException {
+        return new Requester(owner.root).method("POST")
+                .with("body", body)
+                .with("in_reply_to", comment.getId())
+                .to(getApiRoute() + "/comments", GHPullRequestReviewComment.class)
+                .wrapUp(this);
+    }
+
     /**
      * Merge this pull request.
-     *
+     * <p>
      * The equivalent of the big green "Merge pull request" button.
      *
-     * @param msg
-     *      Commit message. If null, the default one will be used.
+     * @param msg Commit message. If null, the default one will be used.
      */
     public void merge(String msg) throws IOException {
-        merge(msg,null);
+        merge(msg, null);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Make this line start at column 5. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AIndentationCheck)